util/tracing/opt_trace.go

```diff
@@ -232,44 +232,48 @@ func (tracer *OptimizeTracer) RecordFinalPlan(final *PlanTrace) {
 
 // PhysicalPlanCostDetail indicates cost detail
 type PhysicalPlanCostDetail struct {
-	id     int
-	tp     string
-	params map[string]interface{}
-	desc   string
+	ID     int                    `json:"id"`
+	TP     string                 `json:"type"`
+	Params map[string]interface{} `json:"params"`
+	Desc   string                 `json:"desc"`
```

Comment:
Why do we export them if we already have setter methods.

Reply:
We export these params in order to transform in json later

Reply:
OK. Then I think we need to check the actual JSON result in the tests in the future.